front/src/modules/companies/table/components/CompanyTable.tsx

```diff
@@ -30,11 +29,20 @@ export function CompanyTable() {
     sortsOrderByScopedSelector,
     TableRecoilScopeContext,
   );
+  const whereFilters = useRecoilScopedValue(
+    filtersWhereScopedSelector,
+    TableRecoilScopeContext,
+  );
+
   const [updateEntityMutation] = useUpdateOneCompanyMutation();
   const upsertEntityTableItem = useUpsertEntityTableItem();
 
   const objectId = 'company';
-  const { handleViewsChange } = useTableViews({ objectId });
+  const { handleViewsChange } = useTableViews({
```

Comment:
this should be renamed to useViews instead of useTableViews.

Reply:
I can rename it to `useViews`, but for now this hook uses `tableViewsState`, `TableRecoilScopeContext` internally... and is only used in tables, so for now it's not really a generic hook. Maybe we could rename it later when it's needed for the board?

Reply:
Just realized `useViewFilters` and `useViewSorts` currently use `TableRecoilScopeContext`, so I renamed `useTableViews` as you suggested. We might want later to pass the relevant data/context as params. Should I rename `useTableViewFields` to `useViewFields` as well?